plugins/services/src/js/components/modals/ServiceRootGroupModal/index.tsx

```diff
@@ -113,7 +150,9 @@ class ServiceRootGroupModal extends React.Component<
       isPending: false,
       expandAdvancedSettings: false,
       data: emptyGroupFormData(),
```

Comment:
if `isEdit` we want this to be null until query in `getGroupFormData` completes, correct?